builders/get-started/networks/layer2/bobabase.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Bobabase
+description: Bobabase is the official testnet for the Boba Layer Two Deployment on Moonbeam. Follow this tutorial to connect to Bobabase.
+---
+
+# Get Started with Bobabase
+
+![Bobabase Banner](/images/builders/get-started/networks/bobabase/bobabase-banner.png)
+
+## Introduction
+Boba is a compute-focused Layer 2 (L2) built on the Optimistic Rollup developed by [Optimism](https://www.optimism.io/){target=_blank}. Boba augments the compute capabilities of EVM-compatible blockchains with a variety of features including [Turing hybrid compute](https://docs.boba.network/turing/turing){target=_blank}. After launching on Ethereum, Boba has brought its Layer 2 scaling solution to Moonbeam. Bobabase is the name of Boba's testnet deployment on Moonbase Alpha. Bobabeam (not yet live) refers to Boba's mainnet deployment on Moonbeam.
+
+## Network Endpoints {: #network-endpoints }
+
+--8<-- 'text/endpoints/bobabase.md'
+
+## Chain ID {: #chain-id } 
+
+Bobabase chain ID is: `1297`, which is `0x511` in hex.
```

Comment:
I think we should add the Bobabase variables such as chain id to variables.yml file